content/en/users/cloud-orchestration/cli/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "IM CLI"
+title: "Infrastructure Manager Command-Line Interface"
```

Comment:
This is too long in the navigation tree. Add a line "linkTitle" with just "Command Line", as you can see here https://docs.egi.eu/users/getting-started/cli/

Reply:
changed.